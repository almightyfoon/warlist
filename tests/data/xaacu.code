c4101b_-0Vkt1v1v1vkpkr2Ikq4i1Zkgkkkg1_ko___-0B0F191h1h1h2Ai22k2k2of0h-h_h-h_bBbB
