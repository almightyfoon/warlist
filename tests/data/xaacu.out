https://conflictchamber.com/#c4101b_-0Vkt1v1v1vkpkr2Ikq4i1Zkgkkkg1_ko___-0B0F191h1h1h2Ai22k2k2of0h-h_h-h_bBbB

Cryx Army - 75 / 75 points
[Theme] Slaughter Fleet Raiders

[Skarre 3] Skarre, Admiral of the Black Fleet [+27]
 - Slayer [10]
 - Slayer [10]
 - Slayer [10]
 - Satyxis Blood Priestess [4]
Axiara Wraithblade [0(6)]
General Gerlak Slaughterborn [0(6)]
Misery Cage [2]
Saxon Orrik [4]
Black Ogrun Boarding Party (max) [11]
Blighted Trollkin Marauders (max) [15]
 - Jussika Bloodtongue [0(5)]
Blighted Trollkin Marauders (max) [15]
Bloodgorgers (max) [15]
Scharde Dirge Seers [6]

Cryx Army - 75 / 75 points
[Theme] Dark Host

[Skarre 1] Pirate Queen Skarre [+28]
 - Deathripper [6]
 - Stalker [8]
 - Stalker [8]
 - Stalker [8]
 - Skarlock Thrall [4]
Bane Lord Tartarus [0(6)]
Machine Wraith [2]
Machine Wraith [2]
Scrap Thrall (3) [2]
Soul Trapper [1]
Bane Warriors (max) [16]
 - Bane Warrior Officer & Standard [0(5)]
Bane Warriors (max) [16]
 - Bane Warrior Officer & Standard [0(5)]
Wraith Engine [15]
Wraith Engine [15]
